internal/operators/cnv/cnv_operator.go

```diff
@@ -128,8 +128,9 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 	return &Operator
 }
 
-// GetHostRequirementsForRole provides operator's requirements towards host in a given role
-func (o *operator) GetHostRequirementsForRole(_ context.Context, _ *common.Cluster, role models.HostRole) (*models.ClusterHostRequirementsDetails, error) {
+// GetHostRequirements provides operator's requirements towards the host
+func (o *operator) GetHostRequirements(_ context.Context, _ *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
+	role := host.Role
 	switch role {
```

Comment:
nitpick :smile: 

```suggestion
	switch host.Role {
```

Reply:
That was deliberate, as the role is used in one more place. I don't feel especially attached to that style, though:)